app/controllers/concerns/account_concern.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+module AccountConcern
+  extend ActiveSupport::Concern
+
+  ACCOUNT_SESSION_REQUEST_HEADER_NAME = "HTTP_GOVUK_ACCOUNT_SESSION"
+  ACCOUNT_SESSION_RESPONSE_HEADER_NAME = "GOVUK-Account-Session"
+  ACCOUNT_END_SESSION_RESPONSE_HEADER_NAME = "HTTP_GOVUK_ACCOUNT_END_SESSION"
```

Comment:
The response header name is `GOVUK-Account-End-Session`, not `HTTP_GOVUK_ACCOUNT_END_SESSION`